playbooks/pulp.yml

```diff
@@ -60,7 +60,7 @@
   roles:
     - common
     - postgres
-    - pulp-web
+    - { role: "pulp-web", when: [ ingress_type | lower != 'route' ] }
```

Comment:
why this change is possible only with this ingress type? or we have just not yet tested other types?

Reply:
For now, in this PR/issue, we are just focusing on `Routes`. If everything works as expected we are planning to open a new issue to test and configure the same design using `Ingress`.